packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -859,6 +887,29 @@ export default class New extends Component {
                       />
                     </fieldset>
                   )}
+                  {type === 'zfs' && (
+                    <fieldset>
+                      <label htmlFor='selectSrLocation'>
+                        {_('srLocation')}
+                      </label>
+                      <select
```

Comment:
Use [`form/select`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/form/select.js) and `optionRenderer` props to display `option` as you want.

Reply:
What is the problem with `html` `select` ?

Reply:
No problem. `form\select` readable, easy to use, `options` props take an array, and more homogeneous in `XO`.

Reply:
@pdonias what's your opinion on this? Should we always use `form/Select`, even for simple cases?